flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisConfigOptions.java

```diff
@@ -307,7 +307,16 @@ public class DorisConfigOptions {
                     .booleanType()
                     .defaultValue(false)
                     .withDescription("Whether to use buffer cache for breakpoint resume");
-
+    public static final ConfigOption<Boolean> USE_FLIGHT_SQL =
+            ConfigOptions.key("use.flight.sql")
```

Comment:
The style of configuration items can be unified, such as source.use-flight-sql
source.flight-sql-port